src/ledger/LedgerDelta.cpp

```diff
@@ -417,13 +417,8 @@ LedgerDelta::markMeters(Application& app) const
 }
 
 void
-LedgerDelta::checkAgainstDatabase(Application& app) const
+LedgerDelta::checkAgainstDatabase(Database& db) const
 {
-    if (!app.getConfig().PARANOID_MODE)
```

Comment:
Why did you remove this check? If you want this mode in the tests just set it in the config.